CleanSwift/CleanSwift/Astronomy/AstronomyViewController.swift

```diff
@@ -0,0 +1,188 @@
+    private func createTableView() {
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.register(AstronomyViewCell.self, forCellReuseIdentifier: Constants.cellIdentifier)
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.dataSource = self
+        tableView.delegate = self
+        view.addSubview(tableView)
+        createTableViewAnchor()
+    }
+
+    private func createDeepLinkButton() {
+        deepLinkButton.translatesAutoresizingMaskIntoConstraints = false
+        deepLinkButton.setImage(UIImage(named: I.telegram.name), for: .normal)
+        deepLinkButton.layer.cornerRadius = 20
+        deepLinkButton.backgroundColor = .secondarySystemBackground
+        msWordButton.tag = 0
+        deepLinkButton.addTarget(self, action: #selector(tapAction), for: .touchUpInside)
+        view.addSubview(deepLinkButton)
+        setupDeepLinkButtonAnchor()
+    }
+
+    private func createMSWordButton() {
+        msWordButton.translatesAutoresizingMaskIntoConstraints = false
+        msWordButton.setImage(UIImage(named: I.msWord.name), for: .normal)
+        msWordButton.layer.cornerRadius = 20
+        msWordButton.backgroundColor = .secondarySystemBackground
+        msWordButton.tag = 1
+        msWordButton.addTarget(self, action: #selector(tapAction), for: .touchUpInside)
+        view.addSubview(msWordButton)
+        setupMSWordButtonAnchor()
+    }
+
+    private func createTableViewAnchor() {
+        tableView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+        tableView.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        tableView.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+        tableView.heightAnchor.constraint(equalToConstant: 600).isActive = true
+    }
+
+    private func setupDeepLinkButtonAnchor() {
+        NSLayoutConstraint.activate([
+            deepLinkButton.topAnchor.constraint(equalTo: tableView.bottomAnchor, constant: 70),
+            deepLinkButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            deepLinkButton.widthAnchor.constraint(equalToConstant: 60),
+            deepLinkButton.heightAnchor.constraint(equalToConstant: 60),
+            deepLinkButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -100)
+        ])
+    }
+
+    private func setupMSWordButtonAnchor() {
+        NSLayoutConstraint.activate([
+            msWordButton.topAnchor.constraint(equalTo: tableView.bottomAnchor, constant: 70),
+            msWordButton.leadingAnchor.constraint(equalTo: deepLinkButton.trailingAnchor, constant: 20),
+            msWordButton.widthAnchor.constraint(equalToConstant: 60),
+            msWordButton.heightAnchor.constraint(equalToConstant: 60),
+            msWordButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -100)
+        ])
+    }
+
+    @objc private func tapAction(_ sender: UIButton) {
```

Comment:
```suggestion
    @objc private func tapAction(_ sender: UIButton) {
```
я бы разнесла все таки в разные методы, хотя бы то что внутри кейсов
и лучше переименовать  tapAction